Sources/ArcGISToolkit/Common/AttachmentsFeatureElementView.swift

```diff
@@ -67,18 +67,33 @@ struct AttachmentsFeatureElementView: View {
     @State private var isExpanded = true
     
     var body: some View {
-        Group {
+        Section {
             switch attachmentModelsState {
             case .notInitialized, .initializing:
                 ProgressView()
                     .padding()
+                    .task {
+                        guard case .notInitialized = attachmentModelsState else { return }
```

Comment:
Since this task should always only run once before the view appears, I wonder if we need this guard. Theoretically, the condition should always be true.

I know this is also on `v.next`. It is probably an optimization that we initially missed, so this might be a good opportunity to fix it.

Reply:
Ok yeah it looks like it could be removed.